torch/jit/__init__.py

```diff
@@ -656,6 +656,25 @@ def __getitem__(self, k):
             raise KeyError(k)
         return self.module._get_parameter(k)
 
+# types that can be constants. If you edit this list,
+# then you also need to edit the handlers in ConstantValue in jit/script/init.cpp
+_constant_types = [bool, float, int, Module, types.FunctionType]
+
+
+def _get_valid_constant(v):
+    if any(isinstance(v, typ) for typ in _constant_types):
+        return v
+    elif isinstance(v, tuple) or isinstance(v, list):
+        return tuple(_get_valid_constant(x) for x in v)
```

Comment:
It would be useful to note that `_constant_types` isn't an exhaustive list, and should not be used to implement checks